.github/workflows/ci.yml

```diff
@@ -118,6 +118,129 @@ jobs:
+        env:
+          TEST_IMAGE_LIST: ${{github.workspace}}/repolist.toml
+          BUSYBOX_TESTING_IMAGE_REF: "k8s.gcr.io/e2e-test-images/busybox:1.29-2"
+          RESOURCE_CONSUMER_TESTING_IMAGE_REF: "k8s.gcr.io/e2e-test-images/resource-consumer:1.10"
+          CGO_ENABLED: 1
+        run: |
+          cat > "${{ env.TEST_IMAGE_LIST }}" << EOF
+          busybox = "${{ env.BUSYBOX_TESTING_IMAGE_REF }}"
+          ResourceConsumer = "${{ env.RESOURCE_CONSUMER_TESTING_IMAGE_REF }}"
+          EOF
+
+          FOCUS="[^TestContainerdRestart$]" make cri-integration
+
+#      Enable these tests once the required JobContainer images are updated.
+# 
+#      - name: Install containerd service
+#        shell: powershell
+#        run: |
+#          mkdir C:\containerd
+#          Set-Content C:/containerd/containerd.toml @"
+#          version = 2
+#          [plugins]
+#              [plugins."io.containerd.grpc.v1.cri".containerd]
+#                default_runtime_name = "runhcs-wcow-process"
+#                disable_snapshot_annotations = false
+#                discard_unpacked_layers = false
+#                ignore_blockio_not_enabled_errors = false
+#                ignore_rdt_not_enabled_errors = false
+#                no_pivot = false
+#                snapshotter = "windows"
+#
+#                [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
+#
+#                [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runhcs-wcow-hypervisor]
+#                  runtime_type = "io.containerd.runhcs.v1"
+#                  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runhcs-wcow-hypervisor.options]
+#                    Debug = true
+#                    DebugType = 2
+#                    SandboxPlatform = "windows/amd64"
+#                    SandboxIsolation = 1
+#
+#                  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runhcs-wcow-process]
+#                    runtime_type = "io.containerd.runhcs.v1"
+#                    pod_annotations = ["microsoft.com/*", "io.microsoft.*" ]
+#                    container_annotations = ["microsoft.com/*", "io.microsoft.*" ]
+#
+#                    [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runhcs-wcow-process.options]
+#          "@
+#
+#          containerd.exe --register-service --log-level=debug --config C:/containerd/containerd.toml --service-name containerd --address //./pipe/containerd-containerd --state C:/ProgramData/containerd/state --root C:/ProgramData/containerd/root --log-file C:/containerd/containerd.log
+#          Set-Service containerd -StartupType Automatic
+#          Start-Service containerd
+#
+#      - name: Build test binary
+#        working-directory: src/github.com/Microsoft/hcsshim/test
+#        shell: powershell
+#        run: |
+#          go test -mod=mod -o "${{ github.workspace }}/bin/cri-containerd.test.exe" -gcflags=all=-d=checkptr -c ./cri-containerd/ -tags functional
```

Comment:
Same as above, can we use the `test/cri-containerd.test.exe` binary from the `test` here?